pdm/pep517/wheel.py

```diff
@@ -169,15 +169,21 @@ def _build(self, wheel):
         lib_dir = next(build_dir.glob("lib.*"), None)
         if not lib_dir:
             return
+
+        _, excludes = self._get_include_and_exclude_paths(for_sdist=False)
         for pkg in lib_dir.glob("**/*"):
             if pkg.is_dir():
                 continue
 
-            rel_path = pkg.relative_to(lib_dir).as_posix()
+            whl_path = pkg.relative_to(lib_dir).as_posix()
+            rel_path = pkg.relative_to(self.location)
+            if self._is_excluded(rel_path, excludes):
```

Comment:
`build` dir is a temp dir and people are not like to write `excludes = ["build/*"]`

Reply:
A better solution is to consider `package-dir` here